nativelib/src/main/scala-2/scala/scalanative/unsafe/UnsafePackageCompat.scala

```diff
@@ -79,6 +79,19 @@ private object MacroImpl {
   import scala.reflect.macros.blackbox.Context
 
   def alloc1[T: c.WeakTypeTag](c: Context)(tag: c.Tree, z: c.Tree): c.Tree = {
+    c.warning(
```

Comment:
Is there by any chance a `c.deprecationWarning` that we could use? If yes, that would take into account the `-deprecation` flag, which would be awesome.

Reply:
I've not found any mentions of such a method in either black-box or white-box macros, there are only 3 methods that can b used for logging: `echo`, `warning`, `abort`